### ISSUE TITLE
Russian laser designators and rangefinders overhaul

### ISSUE THREAD

**Issue by Roman5018** [PygmalionOfCyprus/cmo-db-requests#4319]
### DB Selector

DB3K

### Affected DBID(s)

Aircraft 6957, 6964, 997, 4220, 2604, 2603, 689, 4218, 4219, 827, 1177, 382, 427, 356, 131, 1457, 471, 1735, 4691, 4656, 385, 349, 1890, 4601, 381, 4033, 6727, 570, 379, 2031, 4692, 4655, 352, 6337, 6335, 5649, 5650, 5907, 6467, 6222, 2378, 307, 767, 1892, 6930, 4600, 4262, 6982, 3205, 4430, 133, 434, 1001, 3758, 6984, 4713, 6879, 4714, 6880, 2371, 4230, 6928, 6844, 6864, 6929, 4265, 6983, 6563, 6865, 6881, 6866, 6882, 3321, 4550, 4551, 4231, 2513, 5926, 4445, 6841, 2584, 218, 3015, 2585, 1188, 1187, 2702, 2258, 2237, 78, 7002, 7018, 6842, 2257, 2236, 874, 6958, 5904, 1010, 4213, 2583, 470, 468, 6725, 1743, 600, 1323, 1325, 1327, 6740, 1324, 132, 3759, 1326, 4693, 4468, 3197, 3196

### Summary of Changes

Many Soviet and Russian attack aircraft have incorrect working angles of the laser designator. This applies to the Su-25, Su-17, MiG-27 and Su-24 family aircraft. In the CMO, these aircraft have the ability to provide 360-degree laser targeting. In fact, they had rather narrow viewing angles. For example, the Su-34 has correct angles of +-45 degrees in the front hemisphere. I propose to bring all other aviation laser targeting systems to realism.

Su-25 (Also Su-25SM, Su-25SM2, Su-25K, Su-25KM, Su-25M1, Su-25M1K, Su-25UB, Su-25UBK, Su-25UBM1, Su-25UBM1K, Su-25UTG)
These aircraft had a Klen-PS laser designator with azimuth angles of +-12 degrees and a elevation angle of +6...-30 degrees. That is, in the game they should have the minimum possible viewing angle of 2 "sectors" in the front hemisphere, or 45 degrees, instead of 360, as it is now. In addition, the Klen-PS could not be used to aim laser-guided bombs due to limited downward viewing angles. Only laser-guided missiles, such as the X-29L, should be left in the list of available weapons.

Su-25SM3
This aircraft is equipped with the SOLT-25 optoelectronic system. Despite the fact that the complex is quite modern, has an infrared channel, 16x zoom, automatic tracking, viewing angles remained at the level of Klen-PS. +-15 degrees azimuth and elevation angle of +5...-30 degrees, which is equivalent to the minimum possible two "sectors" in the CMO equal to 45 degrees. Not only the angles of operation of the designator, but also the television and infrared channels of the system should be limited. It also cannot be used to aim laser-guided bombs.

SU-25T (Also Su-25TK, Su-25TM)
It is equipped with the I-251 Shkval system, which has azimuth viewing angles of +-35 degrees and +15...-80 degrees in elevation angle. Now in CMO, the Shkval TV channel has a viewing angle of 2 "sectors" or 45 degrees, while the laser designator operates at 360 degrees. I think that the viewing angle of both systems should be set to 4 "sectors" or 90 degrees in the front hemisphere, since this value is closer to the real 70 degrees. The Shkval also does not have the ability to aim laser-guided bombs.

Su-17M-3 (Also Su-17M-4, Su-22M-3, Su-22M-4, Su-22UM-3K)
It has exactly the same Klen-PS laser designator as the Su-25. Now it works at 360 degrees, it should only work for 2 "sectors" in the front hemisphere or 45 degrees.
UPD. It is also necessary to change the names of the Su-22M-3 and Su-22M-4. In reality, they did not have the letter K in the name.

MiG-27M
It has a Klen-PM laser designator, which is practically no different from the Klen-PS, but for some reason its range in the CMO is 2 times greater, 10 nautical miles instead of 5. I think that the range should be reduced to 5 nautical miles, since open sources report the possibility of laser targeting at a distance of up to 7-8 kilometers. Now it works at 360 degrees, but it should only work for 2 "sectors" in the front hemisphere, or 45 degrees. And it also does not have the ability to use laser-guided bombs.
UPD. The early version of the MiG-27 without the letter M did not have a laser designator at all, only a Fone laser rangefinder (Sensor 1150), the same as the MiG-23BN, which could not be used to aim any laser-guided weapons.

MiG-27K
This aircraft is equipped with the Kaira-23 sighting system. In the CMO, the Kaira TV channel has a viewing angle of 2 "sectors" or 45 degrees, and the laser designator can operate at 360 degrees. In reality, the Kaira-23 had viewing angles of +-35 degrees in azimuth and 0...-160 degrees in elevation. This feature allowed her to use laser-corrected aerial bombs, since the laser beam of target designation could pass into the rear hemisphere  To implement this feature in the CMO, you need to set the viewing angles to 4 "sectors" in the front hemisphere and 4 "sectors" in the rear. You will get an hourglass shape. Hopefully the mechanics of the game allows you to make viewing angles from two parts.

Su-24M (Also Su-24M2, Su-24MK, Su-24MK1, Su-24MK2)
It has a Kaira-24M sighting system, which is practically no different from the Kaira-23. Now the TV channel is working on 2 "sectors" in the front hemisphere, and the laser designator is 360 degrees. There should be 4 "sectors" or 90 degrees in the front hemisphere and 4 "sectors" in the rear. It also has the ability to aim laser-guided bombs.

### Sources

http://lasers.org.ru/forum/threads/Военное-применение-полупроводниковых-лазеров.3105/
https://forum.dcs.world/topic/27144-su-25sm/page/59/
https://ru.m.wikipedia.org/wiki/Шквал_(авиационный_прицельный_комплекс)
https://military.wikireading.ru/huymNMIzH6
https://lektsii.org/12-58363.html

**Comment by FrangibleCover** [comment]
I know you're right, but you might as well include the Russian sources. Translation software is pretty swish these days and it's better to have the paper trail.

**Comment by Roman5018** [comment]
> I know you're right, but you might as well include the Russian sources. Translation software is pretty swish these days and it's better to have the paper trail.

I have added sources. I hope this will help to accept the changes I have proposed.

**Comment by PavelNovak79** [comment]
Minor note:
Late Su-17 export names are Su-22M-3 and Su-22M-4. They don't have "K" in their name. On the other hand there really is Su-22UM-3K but that is because there is also Su-22UM-3 with different engine. The one with "K" in name is more advanced.

Major note:
Original MiG-27 has just Fon laser rangefinder as MiG-23BN and is able to use only command guided missiles like Kh-23 [AS-7] and Kh-25MR [command guided variant of AS-10]. Only later MiG-27M got Klen-PM laser designator.

Otherwise I agree with proposed changes.

**Comment by KLABDB3K** [comment]
Yup... Raised this way back on the old forum,, specifically about the SOLT25 on the Su-25SM?. Plus Notable that even now T220 pod is a mythical beast yet to be seen in the wild....

**Comment by claudejdev** [comment]
Guys, I have some source material for Soviet/Russian equipment, but no time to process it.
Can you reach out to me on Discord?
I'll be happy to share.
@Roman5018 @PavelNovak79 @KLABDB3K

**Comment by Roman5018** [comment]
> Minor note: Late Su-17 export names are Su-22M-3 and Su-22M-4. They don't have "K" in their name. On the other hand there really is Su-22UM-3K but that is because there is also Su-22UM-3 with different engine. The one with "K" in name is more advanced.
> 
> Major note: Original MiG-27 has just Fon laser rangefinder as MiG-23BN and is able to use only command guided missiles like Kh-23 [AS-7] and Kh-25MR [command guided variant of AS-10]. Only later MiG-27M got Klen-PM laser designator.
> 
> Otherwise I agree with proposed changes.

Thanks for your corrections, I made changes to the original post.
